bpf/bpf_sock.c

```diff
@@ -328,9 +298,11 @@ static __always_inline int __sock4_xlate_fwd(struct bpf_sock_addr *ctx,
 	const bool in_hostns = ctx_in_hostns(ctx_full, &id.client_cookie);
 	struct lb4_backend *backend;
 	struct lb4_service *svc;
+	__u16 dst_port = ctx_dst_port(ctx);
+	__u32 dst_ip = ctx->user_ip4;
 	struct lb4_key key = {
-		.address	= ctx->user_ip4,
-		.dport		= ctx_dst_port(ctx),
+		.address	= dst_ip,
+		.dport		= dst_port,
```

Comment:
I wonder, why this and similar changes below were needed?

Reply:
This was just to reuse the variables.

Reply:
Yeah, but they aren't reused :) Anyway, I am ok with this change, maybe it looks better indeed.

Reply:
Both `dst_ip` and `dst_port` are reused when passed to `send_trace_sock_notify`.

Reply:
Ok, I see, it's a later commit (I read changes by commits). Thanks!